### ISSUE TITLE
Refactoring 2: Static web resources

### ISSUE THREAD

**Issue by stlim0730** [stlim0730/farmview#78]
To better organize and serve static files for the two apps (pages and map), this project will have one source (or directory path) of static files. We'll work on the same branch `refactoring`.

`farmview/new_settings.py` on this branch is a template of the settings we will use in the near future (this file will simply replace `farmview/settings.py`). The Django doesn't actually access it for now; It's rather documentation. You can test whether everything is working correctly by temporarily replacing it during the development.

- `BASE_DIR` will have `/static` directory for both apps. They currently have their own static directories. We want to put all the static file resources in one place.
    * Group by resource type (css, js, etc.)
    * 3rd party resources like Cartodb.js, Bootstrap, etc. go into their own subdirectory.
    * 3rd party resources like lightbox2, which come out as packages, go into their own subdirectory
    * `farmview/new_settings.py` has the directory settings we want to use.
    * The structure inside `/static` will look similar to this:

```
/
/static
    /css
        /bootstrap
        style.css (where our custom styles go)
    /font
    /js
        /bootstrap
        /cartodb
        /jquery
    /img
    /lightbox2 (where packaged 3rd party resources go)
```

- In an HTML code, `/static/img/foo.jpg` will automatically generate the correct URL reference. This is what `farmiview/settings.py` does for you. You won't need static template tags, such as `{% load static from staticfiles %}`.
    * During the development, you may want to check if Django can find the file you want using the built-in command line tool below.
```
$ python manage.py findstatic --verbosity 2 /static/foo/bar.jpg
```

- The two apps, map and pages, should share css for consistent HTML styling. They now have similar but different copies of it. This should be manually resolved, but it won't be hard.
    * Modularized css might sound more fascinating, but map / pages is not a good distinction for modules.
- Unused static files should be removed. We have many.

As you might have noticed, this issue requires frequent code-push and communication. One reason is that we're working on the same branch. Keep pushing your code updates with brief explain texts. For example,

> I moved folder A to B. To make C work with B relocated, I changed D in file E.

When you saw any push on this branch, you should pull it (`git pull` on the branch `refactoring`) to be on the same page.

**Comment by stlim0730** [comment]
Yesterday, wifi in my room has been cut at the right (?) moment lol.
If you have any questions or issues, just let me know.

**Comment by sohyeonlee** [comment]
I took a look and cannot find any relevant files to put inside font or lightbox2 folders. Are there other things need to be done?

**Comment by stlim0730** [comment]
@sohyeonlee, thanks for checking in. I think I finished all the stuff needed on this issue.
Merged(#82)
